### PR TITLE
Create application login

### PR DESCRIPTION
**Pull request by andsemenov**

### CODE REVIEW

File: client/src/components/ApplicantLogin.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        localStorage.setItem("user", JSON.stringify(data.user));
+        // document.location.reload();
+        setSuccess(true);
+      })
+      .catch(() => {
+        setError(true);
+        setSuccess(false);
+      });
+  };
+  console.log(success);
```
Comment (llh1) [inline client/src/components/ApplicantLogin.js]: console.log should be removed.

File: client/src/components/ApplicantLogin.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  const loginSubmit = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        console.log(token);
+
+        localStorage.setItem("token", token);
+        localStorage.setItem("user", JSON.stringify(data.user));
+        // document.location.reload();
```
Comment (llh1) [inline client/src/components/ApplicantLogin.js]: Commented code should be removed.

File: client/src/components/ApplicantLogin.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    if (event.target.name === "password") {
+      setPassword(event.target.value);
+    }
+  };
+
+  const loginSubmit = () => {
+    signApi(email, password)
+      .then(data => {
+        const token = data.token;
+        console.log(token);
```
Comment (llh1) [inline client/src/components/ApplicantLogin.js]: Console.log can be removed here

File: client/src/components/ApplicantLogin.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        Login
+      </Button>
+      {error ? <div>Incorrect email or password</div> : null}
+      {success ? <Redirect to="/applicant_create_profile" /> : null}
+    </Form>
+  );
+};
+
+export default ApplicantLogin;
+//<Redirect to="/applicant_create_profile" />
```
Comment (llh1) [inline client/src/components/ApplicantLogin.js]: Commented code can be removed.

File: client/src/components/ApplicantLogin.js (hunk 1 above)
Comment (andsemenov) [reply client/src/components/ApplicantLogin.js]: Thanks for the review. I've fixed the code. Could you review it again, please? Thanks.

**Review by llh1 — APPROVED** [review]

**Pull request merged**